content/en/docs/reference/issues-security/official-cve-feed.md

```diff
@@ -14,18 +15,19 @@ the Kubernetes Security Response Committee. See
 [Kubernetes Security and Disclosure Information](/docs/reference/issues-security/security/)
 for more details.
 
-The Kubernetes project publishes a programmatically accessible
-[JSON Feed](/docs/reference/issues-security/official-cve-feed/index.json) of 
-published security issues. You can access it by executing the following command:
+The Kubernetes project publishes a programmatically accessible feed of published
+security issues in [JSON feed](/docs/reference/issues-security/official-cve-feed/index.json)
+and [RSS feed](/docs/reference/issues-security/official-cve-feed/feed.xml)
+formats. You can access it by executing the following commands:
 
-{{< comment >}}
-`replace` is used to bypass known issue with rendering ">"
-: https://github.com/gohugoio/hugo/issues/7229 in JSON layouts template
-`layouts/_default/cve-feed.json`
-{{< /comment >}}
+For the JSON feed format:
+```shell
+curl -L https://k8s.io/docs/reference/issues-security/official-cve-feed/index.json
+```
 
+For the RSS feed format:
 ```shell
-curl -Lv https://k8s.io/docs/reference/issues-security/official-cve-feed/index.json
+curl -L https://k8s.io/docs/reference/issues-security/official-cve-feed/feed.xml
 ```
```

Comment:
(Optionally)
Could use [tabs](https://kubernetes.io/docs/contribute/style/hugo-shortcodes/#tabs) here. Maybe something for a follow-up?

Reply:
Oh yeah sure good idea!

Reply:
I added tabs it looks way cleaner like that.